packages/wdio-applitools-service/applitools-service.d.ts

```diff
@@ -19,17 +19,17 @@ interface ApplitoolsConfig {
      * Viewport with which the screenshots should be taken.
      */
     viewport?: {
-        width: number;
-        height: number;
+        width?: number;
+        height?: number;
     };
     /**
      * Use proxy for http/https connections with Applitools.
      */
     proxy?: {
-        url: string;
-        username?: string;
-        password?: string;
-        isHttpOnly?: boolean;
+        url: string | boolean;
```

Comment:
How can this be a boolean?

Reply:
I was trying to follow documentation and source code of applitools at the same time. Honestly, it's a mess 😅  There are several implementations of `setProxy()` method that takes either `ProxySettings` object or a number of arguments. I will this one, thanks!